Makefile

```diff
@@ -13,6 +13,11 @@ CFLAGS += -D ENABLE_SDL
 CFLAGS += `sdl2-config --cflags`
 LDFLAGS += `sdl2-config --libs`
 
+# Whether to enable computed goto in riscv.c
+ifeq ("$(NO_COMPUTED_GOTO)", "")
```

Comment:
The computed-goto should be supported only to clang and gcc. You have to detect the compilers.

Reply:
It is not necessary to check `NO_COMPUTED_GOTO`. Instead, the user can manually set `ENABLE_COMPUTED_GOTO`. You only have to validate its availability.